clang/include/clang/Format/Format.h

```diff
@@ -4682,18 +4693,20 @@ struct FormatStyle {
 
     SpacesInParensCustom()
```

Comment:
Maybe set the default of the webkit style, according to the webkit style guide?

Reply:
Thanks for a good catch! I think more correspondence with [`.clang-format` ](https://github.com/WebKit/WebKit/blob/main/.clang-format) and `basedOnStyle: WebKit`, and WebKit style checker script requires more fixes, I may update other things later ;)